doc/manual/install-language.rst

```diff
@@ -0,0 +1,93 @@
+Appendix: Installing the example languages
+==========================================
+
+DOMjudge default ships with some default languages with a default configuration.
+As you might setup contests with those languages we provide how those languages were
+installed in the past as guideline.
+
+Most of those can be found in the table below as there is a package available to install
+inside the judging chroot. Given that you can install your own chroot we only provide the
+packages for Ubuntu as that is the most used at the moment of writing.
+
+.. list-table:: Packages for languages
+   :header-rows: 1
+
+   * - Language
+     - Ubuntu package
+     - Remarks
+   * - Ada
+     - `gnat`
+     -
+   * - AWK
+     - `mawk`/`gawk`
+     - `mawk` is default installed
+   * - Bash
+     - `bash`
+     - Default installed in the chroot
+   * - C
+     - `gcc`
+     - Default installed in the chroot
+   * - C++
+     - `g++`
+     - Default installed in the chroot
+   * - C#
+     - `mono-mcs`
+     -
+   * - Fortran
+     - `gfortran`
+     -
+   * - Haskell
+     - `ghc`
+     - Currently not working
+   * - Java
+     - `default-jdk-headless`
+     - Default installed in the chroot
+   * - Javascript
+     - `nodejs`
+     -
+   * - Kotlin
+     - `kotlin`
+     -
+   * - Lua
+     - `lua5.4`
+     - Ubuntu does not ship a generic meta package (yet).
+   * - Pascal
+     - `fp-compiler`
+     -
+   * - Perl
+     - `perl-base`
+     - Default installed in the chroot
+   * - POSIX
+     - `dash`
+     - Default installed in the chroot
+   * - Prolog
+     - `swi-prolog-core`
+     -
+   * - Python3
+     - `pypy3`/`python3`
+     - Default installed in the chroot.
+       DOMjudge assumes `pypy3` as it runs generally faster.
+       Consider the `PyPy3 PPA`_ if you need the latest python3 features. PyPy3 does not have 100%
+       compatibility with all non-standard libraries in case this is needed you should reconsider the default
+       CPython implementation.
```

Comment:
```suggestion
       DOMjudge assumes `pypy3` as it runs faster in general.
       Consider the `PyPy3 PPA`_ if you need the latest python3 features. PyPy3 does not have 100%
       compatibility with all non-standard libraries. In case this is needed you should reconsider the default
       CPython implementation.
```